init/permissions/public.js

```diff
@@ -0,0 +1,67 @@
+const initialize = {
+  find: { enabled: true, policy: "" },
+  create: { enabled: true, policy: "" },
+}
+const readOnly = {
+  find: { enabled: true, policy: "" },
+  findOne: { enabled: true, policy: "" },
+}
+
+module.exports = {
```

Comment:
Should we do this to configure permissions in staging/production?

Reply:
Added under `npm run init:permissions`